lib/trento/support/commanded_utils.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Trento.Support.CommandedUtils do
+  @moduledoc false
+  alias Trento.ActivityLog
+
+  def commanded, do: Application.fetch_env!(:trento, Trento.Commanded)[:adapter]
```

Comment:
nitpick:
maybe we could have a `def dispatch` function to simply dispatch, instead of putting this `commanded` thing as a public thing. And `commaned` could be a private function.
At the end, it looks like a "detail".
This way, we would only export "useful" things like `dispatch` and `dispatch_correlated`, etc